src/lambdas/manual-change-detector/index.ts

```diff
@@ -0,0 +1,84 @@
+import SlackWebhook from 'slack-webhook'; //this is the only external dependency
+import IAM from 'aws-sdk/clients/iam';
+import zlib from 'zlib';
+import {
+    AWSEvent,
+    AWSCloudwatchEvent,
+    EventInfo,
+} from "./types";
+
+declare const Buffer;
+
+export async function handler(event: AWSEvent): Promise<void> {
+    const iam: IAM = new IAM();
+    const alias: string = (await iam.listAccountAliases().promise()).AccountAliases[0]; //this is only necessary if you have multiple AWS accounts
+
+    const hookUrl: string = "your-slack-webhook-value";
+    const slack: SlackWebhook = new SlackWebhook(hookUrl);
+
+    const payload: Buffer = new Buffer.from(event.awslogs.data, 'base64');
```

Comment:
`Buffer.from()` is a factory method, so no need for `new`.
```suggestion
    const payload = Buffer.from(event.awslogs.data, 'base64');
```